src/main/java/ca/team2706/frc/robot/config/XboxValue.java

```diff
@@ -1,144 +1,144 @@
-package ca.team2706.frc.robot.config;
-
-import java.util.HashMap;
-
-/**
- * Xbox controller binding information.
- * Contains the link between the Xbox's buttons' port and the NetworkTables key used to describe the action.
- */
-public enum XboxValue {

```

Comment:
This file got readded even though there was no changes made. @KyleRAnderson Do you want to fix this, or is okay since it's only one file?